package/CHANGELOG

```diff
@@ -25,6 +25,7 @@ Fixes
     next (Issue #869)
   * Display of Deprecation warnings doesn't affect other modules anymore (Issue #754)
   * Changed nframes to n_frames in analysis modules for consistency (Issue #890)
+  * Removed call to protected process_selection function in RMSD
```

Comment:
Leave this out from the CHANGELOG. I don't think that this was broeken in 0.15.0 or was it? If it was broken then this would be an entry under fixes, just stating that rms.RMSD was fixed (and reference #897)